app/assets/sass/uswds/_uswds-theme-custom-styles.scss

```diff
@@ -671,3 +671,7 @@ details form {
    border: 1px solid #face00;
    border-radius: 7px;
 }
+
+#guidance-h1 {
```

Comment:
I would change this to a class to make it more reusable, or take a look how we can structure the layout so !important isn't necessary (this might not be that easy, just worth exploring)

Reply:
good catch, I've updated the css so it's not using important